src/app/profile-page/profile-claim/profile-claim.service.ts

```diff
@@ -0,0 +1,70 @@
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { mergeMap, switchMap, take } from 'rxjs/operators';
+import { ConfigurationDataService } from '../../core/data/configuration-data.service';
+import { PaginatedList } from '../../core/data/paginated-list.model';
+import { RemoteData } from '../../core/data/remote-data';
+import { EPerson } from '../../core/eperson/models/eperson.model';
+import { DSpaceObject } from '../../core/shared/dspace-object.model';
+import { SearchService } from '../../core/shared/search/search.service';
+import { hasValue } from '../../shared/empty.util';
+import { PaginatedSearchOptions } from '../../shared/search/models/paginated-search-options.model';
+import { SearchResult } from '../../shared/search/models/search-result.model';
+import { getFirstSucceededRemoteData } from './../../core/shared/operators';
+
+@Injectable()
+export class ProfileClaimService {
```

Comment:
Please add TypeDocs to describe this new service.  All methods in this class need TypeDocs as well.  Finally, it looks like this service is missing specs as I don't see a corresponding `profile-claim.service.spec.ts` file.

Reply:
done